### ISSUE TITLE
auto group (with posting requirements) doesn't seem to apply perms/color

### ISSUE THREAD

**Issue by vendethiel** [MightyGorgon/icy_phoenix#80]
I still need to dig deeper, but I've had that on a production forum

**Comment by vendethiel** [comment]
I just confirmed the bug on the latest version. as indicated, clearing the cache does update the permissions correctly.

**Comment by vendethiel** [comment]
@MightyGorgon did you ever check?

**Comment by MightyGorgon** [comment]
I tried a while ago and it seemed working properly... do you still get the same error on latest revision?

**Comment by vendethiel** [comment]
I need to try again.

More importantly - Do you plan to work on IP again, or not? It seems you've been *really* busy since ~ march, and I want to know if you plan to dedicate more time to IP in the future or not?

**Comment by MightyGorgon** [comment]
This is quite off topic here... anyway, as you know I'm alone working on the project (apart from your help on few things), no one is providing feedback, so I assume that IP is dead... I'll continue working on it, but maybe no more official releases, as it seems to me a huge waste of time.

**Comment by KratorD** [comment]
Sorry for off topic, but... seriously IP is dead? No more official releases? :(

**Comment by MightyGorgon** [comment]
If no one is contributing to a project, the project will die... In the past I used to have more free time than now, so I could work on the code constantly and work on new public release... but now things are different, I have less free time and the process of releasing is time consuming: beta testing, fixing bugs, writing docs, moving files, optimizing things...

So, if no one is co-operating, then YES... there will be no further public releases. I will continue working on it (because I mantain several websites based on IP), but no releasing officially.

**Comment by vendethiel** [comment]
The main thing that "bugged" me was that you had so little time, you didn't commit from march, 11 to september, 15. And the commit was mostly legal stuff (cookies etc, which is great because legally required) + js updates + the recurring first post that was already merged (though, don't get me wrong, paypal block and others are nice).
That's why I felt like you couldn't work more on IP (or maybe didn't want to, because of the missing feedback, etc, as you said), but I admit I've been for a long impressed by the incredibly huge commits you used to make, so I'm biased...

I'd never pretend I'm doing enough or that you **have to** do something. It's all volunteer work :-). But there are things where I need your lights (like https://github.com/MightyGorgon/icy_phoenix/issues/67#issuecomment-62326312 ), and we havn't talked a lot about it. The "notifications" thing you committed looks very promising, but it's still TBD. Anyway, that probably should've came after the new version were released...

**Comment by vendethiel** [comment]
Maybe you were right before and the correct solution is to just create a portal that's as good as IP's for phpBB3, if everything is available "there". But I'm not quite sure it's the case? And I think you said you didn't like IP3?

Anyway, I'll follow your decision.

**Comment by MightyGorgon** [comment]
I don't get you wrong... don't worry... I think I said/wrote a lot of time ago that don't have enough free time to follow this project as I did... plus in the meantime PHP + CSS + JS is a brand new world respect to where IP stands... so honestly I see no future on the package as is. Why someone should bother to use Icy Phoenix instead of WP or phpBB 3 or XenForo? Honestly I doubt that a serious website should today prefer IP.

So again, nothing new... I tried to work again on few things, I tried to pay someone to at least improve the template, but neither by paying money I found someone willing to work on this project... so I really think that I'll be the only user for the next years. :-)

If you can code for the project, let me know your plans.

**Comment by vendethiel** [comment]
I believe IP is still an interesting project, because we have

- one of the best CMS/most configurable (seriously, doing stuff like IP's blocks with other portals is just painful)
- a great forum system (which, admittedly, isn't that "special" nowadays)
- an arcade (even if they grew out of popularity)
- an album
- a user blogs system
- a cash and RPG module
- a downloads module
- a knowledge base module
- a few exras, like sudoku/guestbook/feedback/donations/links

But again. If you think phpBB3.1 has the same features and we should try to work with them, I'd follow you. I just don't think it's the case (and you said you didn't like it either way)

**Comment by MightyGorgon** [comment]
I won't work on phpBB 3.1... I will continue improving IP for my own use, and maybe in the future I'll find a good coder willing to re-design the template in a more modern way, which will allow me to add other features which actually are complex to be implemented.

If you have some time and you can check the current install/update procedure we can at least release current state of the art.

**Comment by vendethiel** [comment]
Okay. I'll check this bug first.

**Comment by vendethiel** [comment]
BTW, I still think "clean name" should be a hidden field... I get why it's there (SEO and everything), but I think most users don't care about that. Maybe make it a switch? Not sure.

Do you want me to create new issues here when I find bugs?

After a little bit of search, I've found that, if I have a category > forum (when you install IP, it's "Test Forum 1"), and I append a new, empty category to said forum ( http://i.imgur.com/AoiizKi.png ), then the forum itself ("Test Forum 1") doesn't show up on the forum.php page (but if I add a forum child to the nested category, it goes back to normal).

**Comment by vendethiel** [comment]
I can confirm that this bug report is not totally fixed. I did get the group color, but I had to manually clear the cache/ folder to get the correct permissions.

**Comment by MightyGorgon** [comment]
Ok, can you fix that or do you need me to look into it?

**Comment by vendethiel** [comment]
I'd guess I need to update [this part](https://github.com/MightyGorgon/icy_phoenix/blob/2ea7de2a8db7992bcbbe8397591ca114892be758/includes/functions_post.php#L582) ?

**Comment by vendethiel** [comment]
Also:

> Do you want me to create new issues here when I find bugs?

**Comment by MightyGorgon** [comment]
I'll have a look when I have some time.

**Comment by vendethiel** [comment]
Thanks! I started working on implementing Google connect. I think I'll be able to get Twitter&Reddit connect at the same without much more effort (leveraging the oauth2 library). Unless it gets crazier at work...

**Comment by MightyGorgon** [comment]
Can you try to do this?

OPEN `includes/class_mcp.php`

FIND
```
			if (!empty($user_cache_refresh))
			{
				empty_cache_folders(USERS_CACHE_FOLDER);
			}
```

ADD BEFORE
```
			empty_cache_folders(SQL_CACHE_FOLDER);

```

**Comment by vendethiel** [comment]
I'll do it this evening.

**Comment by vendethiel** [comment]
Forgot to come back to this -- it does indeed fix the issue.

But clearing the SQL cache everytime a user posts might be a bit too cache-heavy, no?

**Comment by MightyGorgon** [comment]
Unfortunately there is no easiest way for now... anyway we are just emptying the SQL CACHE FOLDER not the whole cache.

**Comment by vendethiel** [comment]
Yes it's true. Did you include the change locally?

**Comment by vendethiel** [comment]
fixed in 1fef340d0b12987d283b1fadc05772f275b3012f
